### DISCUSSION TITLE
Please recommend your favorite beer🍺

### DISCUSSION THREAD

**Discussion by Kuwahara-tkm** [NAIST-SE/AttractingContributors#921] [2023 Attraction Contributions]
I would be happy to learn about any beer (various countries, craft beers and major brands).

Please recommend your favorite beer at https://github.com/Kuwahara-tkm/AttractingContributors-Beer

**Comment by bconfessor** [comment]
I sent you a pull request! 
Could you please tell me your favorite song?? 
https://github.com/bconfessor/FavoriteSong/blob/main/README.md
